janusgraph-es/src/main/java/org/janusgraph/diskstorage/es/ElasticSearchIndex.java

```diff
@@ -252,6 +252,11 @@
                 "this parameter to true but we do recommend to reindex your indexes and don't use this parameter.",
             ConfigOption.Type.MASKABLE, false);
 
+    public static final ConfigOption<Integer> CLIENT_KEEP_ALIVE =
+        new ConfigOption<>(ELASTICSEARCH_NS, "client-keep-alive",
+            "Set a keep-alive timeout (in seconds)",
```

Comment:
probably milliseconds as expected by the apache http client would be more sound

Reply:
Yep. I'll change that